src/vcpkg/base/checks.cpp

```diff
@@ -1,6 +1,7 @@
 #include <vcpkg/base/checks.h>
 #include <vcpkg/base/stringview.h>
 #include <vcpkg/base/system.debug.h>
+#include <vcpkg/base/system.print.h>
```

Comment:
It seems like these files needing to drag in an extra header is a bug? (it suggests that one of the other headers here are missing including one of their dependencies)